### PR TITLE
remove unused RingBufferSize

### PR DESCRIPTION
**Pull request by brycekahle**
### What does this PR do?

remove unused ebpf-manager field `RingBufferSize`

### Motivation

The ebpf-manager did not actually use this field when determining the ring buffer size, but used the `MaxEntries` instead.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: pkg/network/tracer/connection/util/conn_tracer.go [hunk 1]
```diff
@@ -93,8 +93,6 @@ func SetupHandler(eventHandler ebpf.EventHandler, mgr *ebpf.Manager, cfg *config
 				RecordGetter:     handler.RecordGetter,
 				RecordHandler:    handler.RecordHandler,
 				TelemetryEnabled: cfg.InternalTelemetryEnabled,
-				// RingBufferSize is not used yet by the manager, we use a map editor to set it in the tracer
```
Comment (akarpz) [inline pkg/network/tracer/connection/util/conn_tracer.go:96]: I think the perfbuffer variant of this type actually does support the size field. Is it very difficult to add it to the ringbuffer so we have consistency?

Comment (brycekahle) [reply pkg/network/tracer/connection/util/conn_tracer.go:96]: With perf buffer: max entries must be <= nCPU, and does not reflect size of buffers
With ring buffer: max entries is the size of the buffer. Having two fields mean the same thing would be confusing.

Comment (brycekahle) [reply pkg/network/tracer/connection/util/conn_tracer.go:96]: We wouldn't know whether to use max entries or RingBufferSize as the true size.

**Review by lebauce — APPROVED** [review]

**Review by guyarb — APPROVED** [review]

File: pkg/network/tracer/connection/util/conn_tracer.go (hunk 1 above)
Comment (brycekahle) [reply pkg/network/tracer/connection/util/conn_tracer.go:96]: @akarpz does that make sense?

Comment (akarpz) [reply pkg/network/tracer/connection/util/conn_tracer.go:96]: sorry, yes. approving now

**Review by akarpz — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Review by gjulianm — APPROVED** [review]

**Pull request merged**